constants/index.ts

```diff
@@ -195,3 +196,24 @@ export const TRIGGER_POST_VARIABLES = [
   '<timestamp>',
   '<txId>'
 ]
+
+export const DEFAULT_PAYBUTTON_TRANSACTIONS_FILE_HEADERS = [
+  'Date', 
+  'Amount', 
+  'Value', 
+  'Rate', 
+  'Transaction Id', 
+  'Paybutton Name'
+];
+
+export const DEFAULT_PAYBUTTON_TRANSACTIONS_FILE_HEADERS_IDS = [
+  'date',
+  'amount',
+  'value',
+  'rate',
+  'transactionId',
+  'paybuttonName'
+];
+
+export const DEFAULT_PAYBUTTON_TRANSACTIONS_FILE_DELIMITER = '\t'
+export const DEFAULT_MAX_RECORDS_FILES = 2000
```

Comment:
Shouldn't it be DEFAULT_MAX_RECORDS_PER_FILE?

BTW, if this is not gonna be changeable, should remove the DEFAULT prefix.